src/types/db.d.ts

```diff
@@ -4,3 +4,10 @@ interface User {
     image: string
     id: string
 }
+
+interface Message{
+    senderId: string
```

Comment:
out of scope right now, but if unread is going to persist between sessions, then it would be stored in the Message object.